lib/valid-properties.js

```diff
@@ -0,0 +1,138 @@
+/**
+* Attribute types.
+*/
+
+var types = {
+    BOOLEAN: 1,
+    OVERLOADED_BOOLEAN: 2
+};
+
+/**
+* Properties.
+*
+* Taken from https://github.com/facebook/react/blob/847357e42e5267b04dd6e297219eaa125ab2f9f4/src/browser/ui/dom/HTMLDOMPropertyConfig.js
+*
+*/
+
+module.exports = {
```

Comment:
Probably you could just convert this all to a simple array since you don't seem to care about the boolean types.